### PR TITLE
PC-572-Loft Insulation Recommendation fix

### PR DESCRIPTION
**Pull request by CalPinSW**
https://beisdigital.atlassian.net/browse/PC-572
- Update YearBuilt enum to include all dates from EPC API
- Update Tests 
- Update GetBreConstructionDate switch statement

### CODE REVIEW

**Review by davidm-m — APPROVED** [review]
Looks sensible to me!

File: SeaPublicWebsite.BusinessLogic/ExternalServices/Bre/RecommendationService.cs [hunk 1]
```diff
@@ -131,7 +131,7 @@ private static BreRequest CreateRequest(PropertyData propertyData)
             BreFlatLevel? breFlatLevel = GetBreFlatLevel(propertyData.PropertyType.Value, propertyData.FlatType);
 
             string breConstructionDate = GetBreConstructionDate(propertyData.YearBuilt, propertyData.WallConstruction, propertyData.CavityWallsInsulated, propertyData.Epc?.ConstructionAgeBand);
-
+
```
Comment (jamiehumphries) [inline SeaPublicWebsite.BusinessLogic/ExternalServices/Bre/RecommendationService.cs:134]: Please can you make sure not to commit these whitespace errors.

**Comment by jamiehumphries** [comment]
⚠️ I think we need to more carefully consider what happens with existing data.

At the moment, any old enum values in the database will be erroneously mapped to the new values.

**Review by jamiehumphries — CHANGES_REQUESTED** [review]
I'm concerned about what will happen with old data. I don't think this is ready yet.

**Comment by jamiehumphries** [comment]
Superceded by https://github.com/UKGovernmentBEIS/beis-simple-energy-advice-beta/pull/261.

**Comment by jamiehumphries** [comment]
Superceded by https://github.com/UKGovernmentBEIS/beis-simple-energy-advice-beta/pull/261.

**Pull request closed without merging**